templates/art/templates.yaml

```diff
@@ -0,0 +1,158 @@
+dataset: art
+templates:
+  151d0e97-d7d2-47f2-86b4-6777587b16f2: !Template
+    id: 151d0e97-d7d2-47f2-86b4-6777587b16f2
+    jinja: "We know that:\n\n{{ observation_1  | trim('.?!') }},\n\nand:\n\n{{ observation_2\
+      \ }} \n\nWhat is more likely?\n\nFirst option: \n\n{{ hypothesis_1  | trim('.?!')\
+      \ }}, \n\nor second option:\n\n{{ hypothesis_2  | trim('.?!') }}?\n|||\n{{ [hypothesis_1,\
+      \ hypothesis_2][label-1]}}"
+    name: hyp4
+    reference: ''
+  2c74c78c-1757-4236-8925-594bbff9a621: !Template
+    id: 2c74c78c-1757-4236-8925-594bbff9a621
+    jinja: 'Which version is more accurate?
+
+
+      The first one:
+
+
+      {{ hypothesis_2  | trim(''.?!'') }},
+
+
+      or the second one:
+
+
+      {{ hypothesis_1  | trim(''.?!'') }}?
+
+
+      Assuming that:
+
+
+      {{ observation_1 }} {{ observation_2 }}
+
+      |||
+
+      {{ [hypothesis_1, hypothesis_2][label-1] }}'
+    name: hyp5_reversed
+    reference: ''
+  2e360dde-c137-405c-bd8b-9e31c9f2aa8c: !Template
+    id: 2e360dde-c137-405c-bd8b-9e31c9f2aa8c
+    jinja: "Given that: \n\n{{  observation_1   | trim('.?!') }}, \n\nand: \n\n{{\
+      \  observation_2  | trim('.?!') }}, \n\nis it true that:\n\n{{ hypothesis_2\
+      \  | trim('.?!')}}?\n|||\n{{ [\"No\", \"Yes\"][label-1] }}"
+    name: hyp2_1
+    reference: ''
+  43fd9dac-ce01-4d9c-9a03-ae38d98bb5aa: !Template
+    id: 43fd9dac-ce01-4d9c-9a03-ae38d98bb5aa
+    jinja: "Does this sentence: \n\n{{ hypothesis_2  | trim('.?!') }} \n\nexplain\

```

Comment:
nitpick - I think if you're trimming punctuation, you can call it a "statement", not a "sentence?